demos/tagged-image/src/ts/components/taggedimage-example/taggedimage-example.component.ts

```diff
@@ -44,6 +47,18 @@ export class TaggedImageExampleComponent extends Component {
         y: Math.random(),
       })),
     })),
+    reshape: (tags) =>
+      tags.map((tag) => ({
+        ...tag,
+        borderRadius: Math.random() * 70 + "%",
+        color: `rgba(${Math.floor(Math.random() * 255)}, ${Math.floor(
+          Math.random() * 255
+        )}, ${Math.floor(Math.random() * 255)}, ${Math.random() / 3 + 2 / 3})`,
+        ...((size) => ({
+          smallSize: Math.floor(size * (0.25 + Math.random() / 3)) + "px",
+          fullSize: size + "px",
+        }))(24 + Math.floor(Math.random() * 50)),
+      })),
```

Comment:
Ui interessant, sieht gut aus aber was macht sie in Worten ausgedrückt? Magst du das als Kommentar hinzufügen?